server/world/world.go

```diff
@@ -806,6 +816,41 @@ func (w *World) Entities() []Entity {
 	return m
 }
 
+// Sleepers returns a list of all sleeping entities currently added to the World.
+func (w *World) Sleepers() []Sleeper {
+	ent := w.Entities()
+	sleepers := make([]Sleeper, 0, len(ent)/40)
+	for _, e := range ent {
+		if s, ok := e.(Sleeper); ok {
+			sleepers = append(sleepers, s)
+		}
+	}
+	return sleepers
+}
```

Comment:
Is there any reason not to repurpose this method to return only those that are actively sleeping?

Reply:
check here: https://github.com/df-mc/dragonfly/pull/938/files#diff-91f78c90dff5aa1562e35a0f1760be7de0398829c40343a40e1b973555bceb62R26-R42
and here: https://github.com/df-mc/dragonfly/pull/938/files#diff-30888af30ee265b9debd16a1ae7820be78a1f73450bd8378c914d1e45dc6f47bR832-R853